autopush/web/message.py

```diff
@@ -32,8 +33,8 @@ def extract_data(self, req):
         except (InvalidToken, InvalidTokenException):
             raise InvalidRequest("Invalid message ID",
                                  status_code=400)
-        return dict(uaid=notif.uaid,
-                    channel_id=notif.channel_id,
+        return dict(uaid=str(notif.uaid),
```

Comment:
you can actually pass these through, marshmallow's UUID field should be happy taking UUID objects